### ISSUE TITLE
jsdoc/require-description-complete-sentence doesn't work with abbreviations

### ISSUE THREAD

**Issue by snowystinger** [gajus/eslint-plugin-jsdoc#581]
## Expected behavior

The sentence, 'This thing works in a certain way, e.g. in this specific way.', is valid.

## Actual behavior

The sentence, 'This thing works in a certain way, e.g. in this specific way.', is broken up into two 'This thing works in a certain way, e.g.' and 'In this specific way.'

## ESLint Config

rules: {
    'jsdoc/require-description-complete-sentence': ERROR,
}

## ESLint sample

```js
// Format code using Markdown code blocks
```

## Environment

* Node version:  12.12.0
* ESLint version  5.16.0
* `eslint-plugin-jsdoc` version: 27.1.1

**Comment by snowystinger** [comment]
Please disregard, I searched for the wrong thing, abbreviation is what to look for
